app/src/main/java/com/sasfmlzr/filemanager/api/other/Param.java

```diff
@@ -0,0 +1,5 @@
+package com.sasfmlzr.filemanager.api.other;
+/**Contain static variables */
+public class Param {
+    public static int sCountActivity =0;
```

Comment:
static fields is not a correct way of passing data between application components. intent extras or listeners must be used instead